### ISSUE TITLE
Apache Directives to put in config not .htaccess files

### ISSUE THREAD

**Issue by jarofgreen** [OpenACalendar/OpenACalendar-Web-Core#581]
.htaccess files have a performance hit.

Can there be a build script or something to pull all these together and spit out apache config to put directly into config?

**Comment by GregSutcliffe** [comment]
Apache configs can be generated, but it may be easier to simply document the addition to the users vhost.

**Comment by jarofgreen** [comment]
Yes, but then you have config in two places and it's to easy to update one and not the other. So a build script is a little clunky, but I think is the better approach.

**Comment by GregSutcliffe** [comment]
Fair point. I was thinking about starting to write some kind of installer anyway, it's the sort of thing i'm good at. I'll start a new discussion on that though, it's tangential to this.

**Comment by jarofgreen** [comment]
https://github.com/JMB-Technology-Limited/ApacheHtaccessIn

**Comment by jarofgreen** [comment]
That's it deployed on OTC. Leaving open, will make sure it's all grand and deploy on HAC and document later ....
